lib/h3core.js

```diff
@@ -585,6 +597,8 @@ export function h3ToGeoBoundary(h3Index, formatAsGeoJson) {
  * @return {H3Index}          H3 index of parent, or null for invalid input
  */
 export function h3ToParent(h3Index, res) {
+    validateIndex(h3Index);
```

Comment:
I thought we weren't forcing these checks to be called on the functions for those who want the perf boost, and those who are unsure of the cleanliness of their data would know to call them?